api-reference/10 UI Widgets/dxPivotGrid/1 Configuration/export/export.md

```diff
@@ -7,6 +7,437 @@ type: Object
+
+        <!-- tab: app.component.ts -->
+        import { Component } from '@angular/core';
+        import ExcelJS from 'exceljs';
+        import saveAs from 'file-saver';
+        
+        @Component({
+            selector: 'app-root',
+            templateUrl: './app.component.html',
+            styleUrls: ['./app.component.css']
+        })
+        export class AppComponent {
+            // ...
+        }
+
+        <!-- tab: app.module.ts -->
+        import { BrowserModule } from '@angular/platform-browser';
+        import { NgModule } from '@angular/core';
+        import { AppComponent } from './app.component';
+        import { DxPivotGridModule } from 'devextreme-angular';
+
+        @NgModule({
+            declarations: [
+                AppComponent
+            ],
+            imports: [
+                BrowserModule,
+                DxPivotGridModule
+            ],
+            providers: [ ],
+            bootstrap: [AppComponent]
+        })
+        export class AppModule { }
+
+
+    ##### Vue
+
+        <!-- tab: App.vue -->
+        <template>
+            <!-- ... -->
+        </template>
+        <script>
+        import 'devextreme/dist/css/dx.common.css';
+        import 'devextreme/dist/css/dx.light.css';
+
+        import { DxPivotGrid, DxExport } from 'devextreme-vue/pivot-grid';
+        import ExcelJS from 'exceljs';
+        import saveAs from 'file-saver';
+
+        export default {
+            components: {
+                DxPivotGrid,
+                DxExport
+            },
+            // ...
+        }
+        </script>
+
+    ##### React
+
+        <!-- tab: App.js -->
+        import React from 'react';
+        import 'devextreme/dist/css/dx.common.css';
+        import 'devextreme/dist/css/dx.light.css';
+
+        import PivotGrid, { Export } from 'devextreme-react/pivot-grid';
+        import ExcelJS from 'exceljs';
+        import saveAs from 'file-saver';
+
+        export default function App() {
+            // ...
+        }
+
+    ---     
+
+2. **Enable the export UI**   
+    Set the **export**.[enabled](/api-reference/10%20UI%20Widgets/dxPivotGrid/1%20Configuration/export/enabled.md '/Documentation/ApiReference/UI_Widgets/dxPivotGrid/Configuration/export/#enabled') option to **true**. This option enables export for all columns. Set a column's [allowExporting](/api-reference/_hidden/dxPivotGridColumn/allowExporting.md '/Documentation/ApiReference/UI_Widgets/dxPivotGrid/Configuration/columns/#allowExporting') option to **false** to prevent it from being exported:
+
+    ---
+
+    ##### jQuery
+
+        <!--JavaScript-->
+        $(function () {
+            $("#pivotGridContainer").dxPivotGrid({
+                export: {
+                    enabled: true
+                },
+                columns: [{ ...
+                    allowExporting: false
+                }, 
+                    // ...
+                ]
+            });
+        });
+
+        <!--HTML-->
```

Comment:
Since this is a step-by-step instruction, we can skip pieces of code that were already shown on the previous steps.